docs/api/covidcast-signals/dsew-cpr.md

```diff
@@ -17,14 +17,18 @@ grand_parent: COVIDcast Epidata API
 
-This data source provides various COVID-19 related metrics, of which we report hospital admissions. Other sources of hospital admissions data in COVIDcast include [HHS](hhs.md) and [medical insurance claims](hospital-admissions.md). The CPR differs from these sources in that it is part of the public health surveillance stream (like HHS, unlike claims) but is available at a daily-county level (like claims, unlike HHS). CPR hospital admissions figures at the state level and above are meant to match those from HHS, but are known to differ. See the Limitations section for details.
+This data source provides various COVID-19 related metrics, of which we report hospital admissions and vaccinations. For hospital admissions, other sources of data in COVIDcast include [HHS](hhs.md) and [medical insurance claims](hospital-admissions.md).  The CPR differs from these sources in that it is part of the public health surveillance stream (like HHS, unlike claims) but is available at a daily-county level (like claims, unlike HHS). CPR hospital admissions figures at the state level and above are meant to match those from HHS, but are known to differ. See the Limitations section for details. For vaccinations, we have no other sources of data currently.
 
-County, MSA, state, and HHS-level values are pulled directly from CPR; nation-level values are aggregated up from the state level.
+County, MSA, state, and HHS-level values are pulled directly from CPR when available; nation-level values are aggregated up from the state level.
 
 | Signal | Description |
 | --- | --- |
 | `confirmed_admissions_covid_1d_7dav` | Number of adult and pediatric confirmed COVID-19 hospital admissions occurring each day. Smoothed using a 7-day average.  <br/> **Earliest date available:** 2019-12-16 for state, HHS, and nation; 2021-01-06 for MSA and county |
 | `confirmed_admissions_covid_1d_prop_7dav` | Number of adult and pediatric confirmed COVID-19 hospital admissions occurring each day, per 100,000 population. Smoothed using a 7-day average. <br/> **Earliest date available:** 2019-12-16 for state, HHS, and nation; 2021-01-06 for MSA and county |
+| `people_full_vaccinated` | 'People fully vaccinated includes those who have received two doses of the Pfizer-BioNTech or Moderna vaccine and those who have received one dose of the J&J/Janssen vaccine' - from the CPR data dictionary.  <br/> **Earliest date available:** 2021-01-15 at any geo level except CBSA and 2021-04-01 at the CBSA level.|
+| `people_booster_doses` |'The count of people who received a booster dose includes anyone who is fully vaccinated and has received another dose of COVID-19 vaccine since 2021-08-13. This includes people who received booster doses and people who received additional doses.' from the CPR data dictionary <br/> **Earliest date available:** 2021-11-01 for state, HHS, and nation. Not available below state level |
```

Comment:
```suggestion
| `people_booster_doses` |"The count of people who received a booster dose includes anyone who is fully vaccinated and has received another dose of COVID-19 vaccine since 2021-08-13. This includes people who received booster doses and people who received additional doses." - from the CPR data dictionary. <br/> **Earliest date available:** 2021-11-01 for state, HHS, and nation. Not available below state level. |
```